composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=7.4",
-        "auth0/auth0-php": "^8.0@beta",
+        "auth0/auth0-php": "dev-8.x-refactor-constructors",
```

Comment:
We can update the constraint to require beta 2

Reply:
Before the release ?

Reply:
Sure.

Reply:
```suggestion
        "auth0/auth0-php": "^8.0@beta2",
```